src/cli/test/parse-flags.spec.ts

```diff
@@ -30,556 +20,216 @@ describe('parseFlags', () => {
     expect(flags.unknownArgs[2]).toBe('test.spec.ts');
   });
 
-  it('should use cli args, no npm cmds', () => {
+  it('should parse cli for dev server', () => {
     // user command line args
     // $ npm run serve --port 4444
 
     // args.slice(2)
     // [ 'serve', '--address', '127.0.0.1', '--port', '4444' ]
 
-    args.push('serve', '--address', '127.0.0.1', '--port', '4444');
+    const args = ['serve', '--address', '127.0.0.1', '--port', '4444'];
 
-    const flags = parseFlags(args, sys);
+    const flags = parseFlags(args);
     expect(flags.task).toBe('serve');
     expect(flags.address).toBe('127.0.0.1');
     expect(flags.port).toBe(4444);
     expect(flags.knownArgs).toEqual(['--address', '127.0.0.1', '--port', '4444']);
   });
 
-  it('should use cli args first, then npm cmds', () => {
```

Comment:
this test (and the test below) is for the now-removed usage of the deprecated `npm_config_argv`, and is deleted for that reason